EventsExpress.Core/IServices/IEventService.cs

```diff
@@ -24,6 +26,8 @@ public interface IEventService
 
         IEnumerable<EventDto> GetAll(EventFilterViewModel model, out int count);
 
+        IEnumerable<EventDto> GetAllDraftEvents(int page, int pageSize, Guid user_id, out int count);
```

Comment:
Why do you need to pass in the user id? It can be accessed using `IHttpContextAccessor`.